types/json2xls/tsconfig.json

```
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
```

```suggestion
        "module": "node16",
```
your module will work with this backward compatible (for CJS) settings